generate-renode-scripts.py

```diff
@@ -120,13 +120,30 @@ def generate_memory_region(region_descriptor):
         string: repl definition of the memory region
     """
 
-    return """
+    result = ""
+
+    if 'realignment_value' in region_descriptor:
+        result += """
+/* WARNING:
+This memory region's size has been
+realigned to allow to simulate it -
+Renode currently supports memory regions
+of size being a multiple of 0x1000.
+
```

Comment:
Can you also add the original alignment?

Reply:
The original size is printed in the next paragraph - do you want to print the starting offset as well?

Automatic realignment currently do not change the initial offset so it's the same as in the generated repl section, but this might not be obvious to the user.

Reply:
It the offset doesn't change (IE you are only extending memory regions to 0x1000) then you are not realigning?

Reply:
To avoid confusion I (a) added realignment of base address (as it's also required by Renode) and (b) changed messages to use 'realign base address' and 'extend size' phrases.

Reply:
@mithro What do you think about this version?